### DISCUSSION TITLE
Contact github organization (if there is no contact/repo) [to ask for transfer of ownership]

### DISCUSSION THREAD

**Discussion by daKmoR** [community/community#23336] [New to GitHub]
<p>hey,</p>
<p>this organization seems to be quite inactive<br>
<a href="https://github.com/mdjs" rel="nofollow noopener">https://github.com/mdjs</a></p>
<p>We do publish an open source tool called <code>mdjs</code> so if there is any chance to get this name it would be awesome.</p>
<p>Unfortunately, I could not find a way to get in contact with the owner? any way to contact? </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/dakmor">@dakmor</a>,</p>
<p>Thanks for being here! Our support folks can check the username and apply our<a href="https://help.github.com/en/github/site-policy/github-username-policy" rel="nofollow noopener">name squatting</a> policy if applicable. If you haven’t already, please contact us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> with the info so we can look into that further.</p>

**Comment by daKmoR** [comment]
<p>thx - I sent the message :hugs:</p>

**Reply by armanist** [reply]
was this resolved?

**Reply by daKmoR** [reply]
"sort of" - it was "decided" that the organization could not be transferred
